auto_nag/scripts/component.py

```diff
@@ -36,10 +37,18 @@ def columns(self):
     def sort_columns(self):
         return lambda p: (-p[3], -int(p[0]))
 
+    def bughandler(self, bug, data):
+        if bug["id"] in self.cache:
```

Comment:
Can you mention the issue you just filed about using bughandler, so we remember that this line is only needed because we are overriding it

Reply:
Done